gcovr/tests/use-existing/README

```diff
@@ -0,0 +1,5 @@
+A test that verifies coverage when using existing *.gcov coverage files with verbose output.
+
+The second conditional in main() includes lines with 
+(, ) and ; characters.  These need to be treated as 
+covered lines.
```

Comment:
This paragraph is misleading, because it is unrelated to the actual test. And the conditional doesn't contain any commas.

Reply:
README file updated to reflect actual content of this folder